app/Http/Controllers/MessagesController.php

```diff
@@ -92,16 +92,20 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        $competition = Competition::find($competitionId);
+        $competition = null;
```

Comment:
interesting why does that code even run when you are sending welcome emails? it should only run when the `type === 'checkin'`